stubs/freezegun/freezegun/api.pyi

```diff
@@ -37,16 +39,16 @@ class _freeze_time:
     @overload
     def __call__(self, func: type[_T]) -> type[_T]: ...
     @overload
-    def __call__(self, func: Callable[..., Awaitable[_T]]) -> Callable[..., Awaitable[_T]]: ...
+    def __call__(self, func: Callable[_P, Awaitable[_T]]) -> Callable[_P, Awaitable[_T]]: ...
     @overload
-    def __call__(self, func: Callable[..., _T]) -> Callable[..., _T]: ...
+    def __call__(self, func: Callable[_P, _T]) -> Callable[_P, _T]: ...
     def __enter__(self) -> FrozenDateTimeFactory | StepTickTimeFactory: ...
     def __exit__(self, *args: Any) -> None: ...
     def start(self) -> Any: ...
     def stop(self) -> None: ...
     def decorate_class(self, klass: type[_T]) -> _T: ...
     def decorate_coroutine(self, coroutine: _T) -> _T: ...
-    def decorate_callable(self, func: Callable[..., _T]) -> Callable[..., _T]: ...
+    def decorate_callable(self, func: Callable[_P, _T]) -> Callable[_P, _T]: ...
```

Comment:
This isn't correct: if as_arg or as_kwarg is given, it injects an additional arg/kwarg. https://github.com/spulec/freezegun/blob/8994558274161b527ab83a3b13ea5969e9ebd144/freezegun/api.py#L783

This also applies to the `__call__` overload for functions.

Reply:
Okay. Best to leave it until we have `Concatenate` support, then?

Reply:
That's not enough for the kwarg.